templates/block-php.mustache

```diff
@@ -6,13 +6,15 @@
  */
 
 /**
- * Enqueues block assets to apply inside the editor only.
+ * Enqueues all block assets to apply both on the front of your site and in the editor.
```

Comment:
This only registers the assets now, it doesn't enqueue them anymore.

Reply:
`register_block_type` enqueues everything. See: https://github.com/WordPress/gutenberg/pull/4039/files#diff-620130744b6b73b822a3df609671d930R606.

Reply:
Yes, but that is done only indirectly (just like WordPress enqueues everything, at one point). Wouldn't it be more accurate to say something like:
```
Registers all block assets so that they can be enqueued through Gutenberg in the corresponding context.
```

Reply:
Sure, updating.